sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -119,7 +119,8 @@ statement
         (TBLPROPERTIES tableProps=tablePropertyList))*
         (AS? query)?                                                   #createHiveTable
     | CREATE TABLE (IF NOT EXISTS)? target=tableIdentifier
-        LIKE source=tableIdentifier tableProvider? locationSpec?       #createTableLike
+        LIKE source=tableIdentifier
+        (tableProvider | createFileFormat)? locationSpec?              #createTableLike
```

Comment:
Can you update https://github.com/apache/spark/blob/master/docs/sql-migration-guide.md#compatibility-with-apache-hive as well?

Reply:
Hi, CTL in Hive support `[TBLPROPERTIES (property_name=property_value, ...)]` which current is not in Spark. So should I support this in this PR?

Reply:
seems reasonable to me to support TBLPROPERTIES in CREATE TABLE LIKE

Reply:
Sorry for reply late. Now I added.